MyCryptoHQ/MyCrypto
# MC-01-001: Prevent drag and drop

Closes https://github.com/MyCryptoHQ/MyCrypto/issues/2002

### Description

> 

> It was found that MyCrypto Electron desktop application allows dragging and dropping any local HTML files onto the window. Since the file: URL in Electron has the privilege to read any local files or web sites, this leads to leakage of sensitive information if the user drags and drops a crafted local file.
> 
> In case the following HTML file is dropped onto the MyCrypto window, the content of /etc/hosts will be shown on an alert dialog.

     - Cure 53


### Changes

This PR disables support for drag-and-drop as a precautionary measure based on Cure53's report.

### Steps to Test

1. yarn dev:electron
2. attempt to drag and drop an html file into the Desktop application
3. Confirm that you are no longer able to
